barretenberg/cpp/src/barretenberg/goblin/goblin.hpp

```diff
@@ -77,68 +69,6 @@ class GoblinProver {
         commitment_key = bn254_commitment_key ? bn254_commitment_key : nullptr;
         GoblinMockCircuits::perform_op_queue_interactions_for_mock_first_circuit(op_queue);
     }
-    /**
-     * @brief Construct a MegaHonk proof and a merge proof for the present circuit.
-     * @details If there is a previous merge proof, recursively verify it.
-     *
-     * @param circuit_builder
-     */
-    GoblinAccumulationOutput accumulate(MegaBuilder& circuit_builder)
-    {
-        // Complete the circuit logic by recursively verifying previous merge proof if it exists
-        if (merge_proof_exists) {
-            RecursiveMergeVerifier merge_verifier{ &circuit_builder };
-            [[maybe_unused]] auto pairing_points = merge_verifier.verify_proof(merge_proof);
-        }
-
-        // Construct a Honk proof for the main circuit
-        auto proving_key = std::make_shared<MegaDeciderProvingKey>(circuit_builder);
-        MegaProver prover(proving_key);
-        auto ultra_proof = prover.construct_proof();
-        auto verification_key = std::make_shared<VerificationKey>(proving_key->proving_key);
-
-        // Construct and store the merge proof to be recursively verified on the next call to accumulate
-        MergeProver merge_prover{ circuit_builder.op_queue };
-        merge_proof = merge_prover.construct_proof();
-
-        if (!merge_proof_exists) {
-            merge_proof_exists = true;
-        }
-
-        return { ultra_proof, verification_key };
-    };
-
-    /**
-     * @brief Add a recursive merge verifier to input circuit and construct a merge proof for the updated op queue
-     * @details When this method is used, the "prover" functionality of the IVC scheme must be performed explicitly, but
-     * this method has to be called first so that the recursive merge verifier can be "appended" to the circuit being
-     * accumulated
-     *
-     * @param circuit_builder
-     */
-    void merge(MegaBuilder& circuit_builder)
-    {
-        // Append a recursive merge verification of the merge proof
-        if (merge_proof_exists) {
-            [[maybe_unused]] auto pairing_points = verify_merge(circuit_builder, merge_proof);
```

Comment:
Same as above. There is no use case for Goblin appending recursive merge verifiers to circuits.